kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -25,6 +25,11 @@ def check(self, _):
 
         consumer_offsets = {}
 
+        try:
+            self.client.request_metadata_update()
+        except:
+            raise Exception("Unable to connect to the AdminClient. This is likely due to an error in the configuration.")
```

Comment:
To reviewer: the `confluent-kafka-python` library raises an unhelpful 
```
cimpl.KafkaException: KafkaError{code=_TRANSPORT,val=-195,str="Failed to get metadata: Local: Broker transport failure"}
```
which is why this message is included.